config/samples/kamaji_v1alpha1_tenantcontrolplane.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: kamaji.clastix.io/v1alpha1
 kind: TenantControlPlane
 metadata:
-  name: 126-k8s
+  name: k8s-126
```

Comment:
this change has nothing to do with the title of the PR and should have been managed with dedicated issues and PR

Reply:
That's partially correct, although it's been addressed with a separate commit with its own commit subject (8d27a1c8d4935306af0d020797b8a1b443c56017).